### ISSUE TITLE
[Feature Request] Possibility to render dots beneath slider

### ISSUE THREAD

**Issue by tpinne** [YIZHUANG/react-multi-carousel#35]
**Feature request**
It would be nice to have the ability to render the dots beneath the slider and not only over it. This could be easily done via changing CSS if the DOM structure would be slightly changed.

With the following changes it should render exactly the same as now. But makes it possible via CSS to push the dots beneath by removing `position: absolute` from their styles. (I guess, not tested).

```html
<div class="react-multi-carousel">
    <div class="react-multi-carousel-list">
        <ul class="react-multi-carousel-track">
        <!-- ... -->
        </ul>
    </div>
    <ul class="react-multi-carousel-dot-list">
    <!-- ... -->
    </div>
</div>
```

```css
.react-multi-carousel {
    position: relative;
}
```

Currently to achieve this one has to implement the dots logic to place it outside the component because of the needed `overflow: hidden;` on the `.react-multi-carousel-list`.

It's somehow possible by adding `padding-bottom` to `.react-multi-carousel-list`. But then you have to know the height of the dots element. And you have padding if there aren't any dots for whatever reason that may be.

Or maybe we missing something.

**Comment by YIZHUANG** [comment]
The way i do is to add ```padding-top``` or ```padding-bottom``` to the ```.react-multi-carousel-list```, if we are changing the structure, we have to think about how to make the API in a way that's backward compatible because you might just need it the way it is in right now some cases.

**Comment by tpinne** [comment]
An option could be to add a new boolean prop which renders the dot list after `.react-multi-carousel-list` instead of inside and wrapping the render in a `<Fragment></Fragment>`.

Then it would be an opt-in feature and then you can wrap it in a custom `div` to get the positioning as needed.

```jsx
<div className="my-custom-wrapper-with-position-relative">
  <Carousel showDots={true} renderDotsOutside={true} />
</div>
```

**Comment by YIZHUANG** [comment]
@tpinne  Sounds like a good idea, can you make it and put the changes in ```feature/new-dots-mode```branch?

**Comment by tpinne** [comment]
I think next week I can give it a go.

**Comment by tpinne** [comment]
I've created a PR #40 for this since you already released the other changes from the `feature/new-dots-mode` branch.

**Comment by YIZHUANG** [comment]
Thanks, it's now released in v2.10
